src/app/[locale]/incident/add/components/IncidentQuestionsLocationForm.tsx

```diff
@@ -92,12 +100,31 @@ export const IncidentQuestionsLocationForm = () => {
       extra_properties: answers,
     })
 
-    setLastCompletedStep(2)
+    addVisitedStep(FormStep.STEP_2_ADD)
     addOneStep()
-
-    router.push('/incident/contact')
+    router.push(steps[FormStep.STEP_3_CONTACT])
   }
 
+  useEffect(() => {
+    if (navToSummary) {
+      // todo: finish update form when component it fixed
+      // updateForm({})
+
+      goToStep(FormStep.STEP_4_SUMMARY)
+      router.push(steps[FormStep.STEP_4_SUMMARY])
+      onNavToSummary(false)
```

Comment:
Hiervoor weer dezelfde feedback als hier boven; ik zou de functie ```onNavToSummary``` veranderen van naam naar ```setNavToSummary```. Dit lijkt meer in lijn met react conventies op het gebied van naming.